pkg/cmd/pulumi/replay_events.go

```diff
@@ -43,6 +43,7 @@ func newReplayEventsCmd() *cobra.Command {
 	var debug bool
 
 	var delay time.Duration
+	var period time.Duration
```

Comment:
In the spirit of getting small changes in do you want to raise PR with just the changes for this file, and we can merge that in like right away and it then cleans it up from all these others PRs.

Reply:
Will do, yep

Reply:
Ah, that's part of https://github.com/pulumi/pulumi/pull/11199. That's going to be merged with those changes.